load_uploaded_files.json

```diff
@@ -0,0 +1,65 @@
+{
+    "id": "load_uploaded_files",
+    "summary": "Load files from the user workspace.",
+    "description": "Loads user-uploaded files from the local user workspace / data store. The files must have been stored by the authenticated user on the back-end currently connected to.",
```

Comment:
The process name, description, ...  talk about plural "files", while a lot of use cases will probably be about single files (e.g. a polygon masking). An even in the multiple file case I think it is worth mentioning in the description that all the data is  combined and returned as a single data cube.